diff --git a/lua/dapui/client/lib.lua b/lua/dapui/client/lib.lua
index d1148ae..29c2d67 100644
--- a/lua/dapui/client/lib.lua
+++ b/lua/dapui/client/lib.lua
@@ -30,7 +30,7 @@ return function(client)
         return util.open_buf(buf, line, column)
       end
 
-      if not source.path or not vim.uv.fs_stat(source.path) then
+      if not source.path then
         util.notify("No source available for frame", vim.log.levels.WARN)
         return
       end
